scipy/fft/_realtransforms.py

```diff
@@ -1,15 +1,34 @@
-from ._basic import _dispatch
-from scipy._lib.uarray import Dispatchable
+from scipy._lib._array_api import array_namespace
+from . import _realtransforms_uarray
 import numpy as np
 
 __all__ = ['dct', 'idct', 'dst', 'idst', 'dctn', 'idctn', 'dstn', 'idstn']
 
 
-@_dispatch
+def _execute_1D(uarray_func, x, type, n, axis, norm,
+                overwrite_x, workers, orthogonalize):
+    xp = array_namespace(x)
+    x = np.asarray(x)
+    y = uarray_func(x, type=type, n=n, axis=axis, norm=norm,
+                    overwrite_x=overwrite_x, workers=workers,
+                    orthogonalize=orthogonalize)
+    return xp.asarray(y)
+
+
+def _execute_nD(uarray_func, x, type, s, axes, norm, 
```

Comment:
Does this need the same simplification to `_execute(...)` as in `_basic.py`, or does that not work for some reason?